system/datatypes/peer_reviewed_article.php

```diff
@@ -1048,12 +1051,27 @@ private function getCommentFormHtml($type, DocPage $discussion = null) {
 		$comment = __('art-comment');
 		$new = $discussion === null;
 		$commentFieldName = self::FIELD_NAME_DISCUSSION_COMMENT . '/' . ($new ? 'new_' . $type : $discussion->getId());
-		$html = <<<EOF
+		if (strpos($commentFieldName, 'new_review') > 0) {
+			$_discussionLabel = __('art-start-discussion');
+			$_discussionShortTitle = __('art-title');
+
+			$html = <<<EOF
 			<div class="new-comment $type">
 			<div>
-				<strong><label for="$commentFieldName"> $comment </label></strong><textarea name="$commentFieldName" id="$commentFieldName" cols="36" rows="4"></textarea>
-			</div>
+				<strong><label for="$commentFieldName">$_discussionLabel</label></strong>
+				$_discussionShortTitle<br><input type="text" name="subject"><br>
+				$comment<br>
+				<textarea name="$commentFieldName" id="$commentFieldName" cols="36" rows="4"></textarea>
+EOF;
+}
+else {
+	$html = <<<EOF
+	<div class="new-comment $type">
+	<div>
+		<strong><label for="$commentFieldName"> $comment </label></strong><textarea name="$commentFieldName" id="$commentFieldName" cols="36" rows="4"></textarea>
+	</div>
 EOF;
+}
```

Comment:
The indentation of this else block seems wrong? I suspect you aligned it with the EOF, but it's probably better to align it with the original if?